cl/users/tasks.py

```diff
@@ -78,7 +79,7 @@ def update_moosend_subscription(self: Task, email: str, action: str) -> None:
 def send_failed_email(
     self: Task,
     failed_pk: int,
-) -> int:
+) -> None:
```

Comment:
This indicates that this file isn't getting processed via mypy. Take a look in lint.yml and see if adding this file (or the entire user module) isn't too difficult. We're doing this incrementally, and when we add new files we should try to remember to add them to CI.

Reply:
I've added cl.users.tasks.py into lint.yml, I tested to add the whole user module but seems there are many issues to solve before, so I could look into it later if it's ok.